tests/gordo_components/builder/test_azure_utils.py

```diff
@@ -0,0 +1,338 @@
+from datetime import datetime
+import pytest
+from pytz import UTC
+from unittest import mock
+import mlflow
+from mlflow.entities import Metric, Param
+from tempfile import TemporaryDirectory
+
+from gordo_components.builder.azure_utils import (
+    _validate_dict,
+    MlflowClient,
+    get_mlflow_client,
+    get_run_id,
+    get_loggables,
+    _is_number,
+    _datetime_to_ms_since_epoch,
+    get_batch_kwargs,
+    get_spauth_kwargs,
+)
```

Comment:
I suggest `from gordo_components.builder import azure_utils as au` or something. All these free functions floating around, and would save 10 lines. ;-)